rosco-core/src/main/groovy/com/netflix/spinnaker/rosco/providers/aws/AWSBakeHandler.groovy

```diff
@@ -53,8 +53,13 @@ public class AWSBakeHandler extends CloudProviderBakeHandler {
     }
 
     // TODO(duftler): Work through definition of uniqueness.
+    // TODO(duftler): Include aws_associate_public_ip_address and enhanced_networking in key?
```

Comment:
aws_associate_public_ip_address is a packer setting on how it connects to the instance in order to do snapshotting when producing the AMI, it has not effect on the resulting image

enhanced_networking should be in the key because that is an attribute of the resulting image